### DISCUSSION TITLE
Aquila G32 or N32 not supported??

### DISCUSSION THREAD

**Discussion by classicrocker883** [mriscoc/Ender3V2S1#340] [Q&A]
**Describe the bug**
Tried the environment [env] and `MOTHERBOARD` for the VoxelabAquila and it doesn't actually work. 

**To Reproduce**
1. in platform.ini change env to GD32F103RE
2. in configuration.h change MB to VOXELAB_AQUILA 

**Expected behavior**
It compiles fine, but when flashing FW it only goes as far as bootscreen. mainscreen doesn't load. 

**Additional context**
I have tried different DWIN types, ProUI, JyersUI, Creality DWIN, and copying over Alex's alexqzd DWIN files. 

I think it may have to do with the stm32f1.ini /-maple.ini or stm32 common.ini

i can see the N32 isn't yet supported but the G32 is but it doesn't actually work.

**Comment by mriscoc** [comment]
Current support for Aquila G32 printers is experimental using the configurator app: https://github.com/mriscoc/Special_Configurations/releases/tag/Aquila
![image](https://user-images.githubusercontent.com/2745567/187774542-8d90fd43-0656-4620-8b09-e4c614804b7d.png)
Try with the Maple environment.

**Reply by JEBurroughs** [reply]
I have a couple of G32 boards and I tried flashing one before I ripped it out and put a Creality 4.2.7 board in. It acted like it flashed but I just got a blank screen with the embedded stuff. That was using both kinds of Display Types and the Display Firmware you pointed to. But since they were going bye bye anyway I didn't pursue it any further. I don't have the first clue about compiling so I had just used the UBL BLTouch version you had posted.
